gulpfile.js

```diff
@@ -129,29 +146,33 @@ for (var dependencyId in frontendDependencies) {
   }
 }
 gulp.task('generateCss', function() {
-  gulp.src(cssFilesPath)
-    .pipe(isMinificationNeeded ? minifyCss() : gulpUtil.noop())
-    .pipe(concat('third_party.css'))
-    .pipe(gulp.dest(generatedCssTargetDir));
+  if (requiresFilesExists(cssFilesPath)) {
```

Comment:
How about

   requireFilesExist(cssFilepaths);
   gulp.src(...)
     .pipe(...)
     ...

and basically having requireFilesExist() raise a terminating error if there are any problems?